lib/happo/public/HappoDiffs.jsx

```diff
@@ -80,10 +90,34 @@ function HappoNewImages({ images }) {
     </div>
   );
 }
+HappoNewImages.propTypes = {
+  images: PropTypes.arrayOf(PropTypes.shape(imageShape)),
+};
+
+function SelectedView({ image, selectedView }) {
```

Comment:
I've been using "Happo" as a prefix to all of these. Don't exactly know why, but it might be worth being consistent.